client/app/scripts/superdesk-users/users.js

```diff
@@ -1297,6 +1297,15 @@
                     scope.selectUser = function(user) {
                         return '@' + user.username;
                     };
+
+                    scope.$watchCollection(
```

Comment:
not sure it's a good idea to use jquery in a watcher, is there maybe some js event it could listen to?